lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -310,6 +310,18 @@ def post(self, request):
                     (UserProfile.objects.all().iterator()))
             return self.return_csv('users_{0}.csv'.format(
                 request.META['SERVER_NAME']), header, data)
+
+        elif action == 'download_subscription_emails':
+            from .models import PlatformNewsSubscriptionEmail
```

Comment:
Can we avoid local import here?

Reply:
yeap, thanks! I just forgot to move it. Fixed